Ports/SDL2/package.sh

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port='SDL2'
-version='2.28.2'
+version='2.30.5'
 useconfigure='true'
 files=(
-    "https://github.com/libsdl-org/SDL/releases/download/release-${version}/SDL2-${version}.tar.gz#64b1102fa22093515b02ef33dd8739dee1ba57e9dbba6a092942b8bbed1a1c5e"
+    "https://github.com/libsdl-org/SDL/releases/download/release-${version}/SDL2-${version}.tar.gz#f374f3fa29c37dfcc20822d4a7d7dc57e58924d1a5f2ad511bfab4c8193de63b"
 )
 configopts=(
     "-DCMAKE_CXX_FLAGS=-I${SERENITY_BUILD_DIR}/Root/usr/include/Services/ -I${SERENITY_BUILD_DIR}/Root/usr/include/Userland/Services/"
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
     "-DPULSEAUDIO=OFF"
     "-DJACK=OFF"
     "-DSDL_LIBSAMPLERATE=OFF" # Disabled to prevent potential collision with host libsamplerate
-    "-DEXTRA_LDFLAGS=-laudio;-liconv;-ldl"
+    "-DEXTRA_LDFLAGS=-L${SERENITY_INSTALL_ROOT}/usr/local/lib;-laudio;-liconv;-ldl"
```

Comment:
This should no longer be needed once #24893 is in.

Reply:
Yes, that looks to be a better solution. I'll drop e128d8e when #24893 is merged.

Reply:
It's merged now 🙂